### DISCUSSION TITLE
Teslim hatası

### DISCUSSION THREAD

**Discussion by Abdullah-Alsabouni** [19/classroom#28] [Anonim] [answered]
![Ekran görüntüsü 2022-03-14 165238](https://user-images.githubusercontent.com/98663047/158186101-5f73d724-4851-4142-9d4b-321cd5d66593.png)
Ödevi teslim ettikten sonra böyle bir şeyle karşılaşıyorum.
![Ekran görüntüsü 2022-03-14 165643](https://user-images.githubusercontent.com/98663047/158186942-38790627-490d-4f81-a287-2d2e2826ec6f.png)

**Comment by gulcanyildiz** [comment]
Şu an 50 puan alabiliyorsun. Ödevin Correct? aşaması doğru değil. #17 ye benzer bir sorun var. 100 puan için programında güncelleme yapıp commit etmelisin.

**Reply by Abdullah-Alsabouni** [reply]
Aynı kodu normal terminal'de derlediğim'de istenenin aynısını veriyor.
![Ekran görüntüsü 2022-03-14 185325](https://user-images.githubusercontent.com/98663047/158211600-41a31605-9bf3-4fb6-be85-0bd89bf60d06.png)
![Ekran görüntüsü 2022-03-14 185601](https://user-images.githubusercontent.com/98663047/158211624-94312c5d-0408-46d0-a801-1fa779b9eab1.png)

**Reply by Abdullah-Alsabouni** [reply]
Ödev sisteminde derlediğimde farklı çıkıyor.

**Comment by roktas** [comment, marked as answer]
@abdullah012345678910

1. Buraya bu kadar ekran görüntüleriyle emek sarfedeceğine sana yapılan yönlendirmeyi (#17) anla ve git dikkatli şekilde kodunu incele/düzelt (ben hatanı görüyorum, bulmak sana kalmış)

2. Gönderdiğin ekran çıktılarında tutarsız bir şey yok, `lets` doğru çalışıyor, ama hadi bir an için hatalı olsun.  `lets` size sunduğum bir "elverişililik" aracı ve şimdiden bunu sunduğuma pişman oldum.  **`lets`'in ne sonuç verdiği önemli değil, Github tarafı ne sonuç veriyorsa ona bakacaksın.**

Yoruyorsunuz beni...

**Comment by roktas** [comment]
@abdullah012345678910 Hesabın Github'la link edilmemiş.  Numaranı ve Ad Soyadını yazar mısın buraya, ekleyelim.  Aksi halde puan alamayacaksın ödevden.
